docs/core/extensions/snippets/logging/getting-started-logger-message/Program.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Microsoft.Extensions.Logging;
+
+internal partial class Program
+{
+    static void Main(string[] args)
+    {
+        using ILoggerFactory factory = LoggerFactory.Create(builder => builder.AddConsole());
+        ILogger logger = factory.CreateLogger("Program");
+        LogStartupMessage(logger, "fun");
+    }
+
+    [LoggerMessage(Level = LogLevel.Information, Message = "Hello World! Logging is {description}.")]
```

Comment:
```suggestion
    [LoggerMessage(LogLevel.Information, "Hello World! Logging is {description}.")]
```

Reply:
I think this one relies on a new constructor that is only present in .NET 8. I left it out since the samples don't (yet) depend on .NET 8